vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -204,79 +204,64 @@
         </Property>
       </ComplexType>
 
+      <TypeDefinition Name="HierarchyReference" UnderlyingType="Edm.String">
+        <Annotation Term="Core.Description" String="Annotation path that references a [`RecursiveHierarchy`](#RecursiveHierarchy) annotation"/>
+        <Annotation Term="Core.LongDescription">
+          <String>Functions defined in this vocabulary have a parameter of this type. It specifies the hierarchy that is evaluated by the function.
+          The annotation path is evaluated starting at the entity set of the function's binding parameter,
+          so it can have the form `@Aggregation.RecursiveHierarchy#qualifier`.

```

Comment:
Or would the qualifier alone be sufficient, like in the `ancestors` transformation?